# This patch prevents a crash in gpg-agent.
# If gettext isn't used until a fork, it could crash when it is used.


--- a/agent/gpg-agent.c
+++ b/agent/gpg-agent.c
@@ -1355,6 +1355,7 @@ main (int argc, char **argv )
     }
 
   set_debug ();
+  _("");
 
   if (atexit (cleanup))
     {
